python/pyproject.toml

```diff
@@ -1,9 +1,10 @@
 [tool.poetry]
 name = "magika"
-version = "0.5.2-dev"
-description = "A tool to determine the content type of a file with deep-learning"
-authors = ["Yanick Fratantonio <yanickf@google.com>"]
+version = "0.6.0-dev"
+description = "A tool to determine the content type of a file with deep learning"
+authors = ["Magika Developers <magika-dev@google.com>"]
```

Comment:
Is this something we want to do in Rust too?

Reply:
I would? That's so that everyone is in the loop, having myself as the only POC was not intended for python :-)

Reply:
Sounds good, I'll do that.